src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XElement.cs

```diff
@@ -0,0 +1,56 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Linq
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_XElement
+    {
+        private XElement _element;
+
+        [Benchmark]
+        public XElement CreateElement() => new XElement("Root", "text node");
+
+        [Benchmark(OperationsPerInvoke = 8)]
+        public XElement CreateWithElements()
+        {
+            XElement doc = new XElement("Root",
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three")
+                );
+
+            return doc;
+        }
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _element = new XElement("Root",
+                new XAttribute("id", 123),
+                new XElement("Child1", 1),
+                new XElement("Child2", 2),
+                new XElement("Child3", 3),
+                new XElement("Child4", 4),
+                new XElement("Child5", 5)
+            );
+        }
+
+        [Benchmark]
+        public XElement GetElement() => _element.Element("Child4");
+        
+        [Benchmark]
+        public XAttribute GetAttribute() => _element.Attribute("id");
+
+        [Benchmark]
+        public string GetValue() => _element.Value;
+
+        [Benchmark]
+        public XName GetXName() => _element.Name;
```

Comment:
I've run these benchmarks and it looks like `.Name` just returns a field value and does not perform any actual work:

|             Method |       Mean |
|------------------- |-----------:|
|      CreateElement | 48.9110 ns |
| CreateWithElements | 87.9020 ns |
|         GetElement | 40.9684 ns |
|       GetAttribute | 30.7949 ns |
|           GetValue | 54.9559 ns |
|           GetXName |  0.0000 ns |

Please remove the `GetXName` benchmark